portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,37 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body{
+    color:white;
+    background-color:black;
+}
```

Comment:
A couple of suggestions.

1. After all of your css definitions i would add a new line to make it easier to read. 

Example
```
body {
  color: white;
}

.bleh {
  width: 100%;
}

...
```

2. After each key I would add a space. It can be a bit hard to read when its like `color:white;`

Example
```
  color: white;
  width: 100%;
...
```

3. I would add a space after the class name or id

Example

```
#headerText {

}
```